include/mc_rbdyn/lipm_stabilizer/StabilizerConfiguration.h

```diff
@@ -132,18 +132,24 @@ struct DCMBiasEstimatorConfiguration
   double biasDriftPerSecondStd = 0.02;
   /// Maximum bias in the sagital and lateral directions [m]
   Eigen::Vector2d biasLimit = {0.02, 0.02};
+  /// Maximum bias in the sagital and lateral directions used to correct the CoM, should be smaller than bisaLimit [m]
+  Eigen::Vector2d comBiasLimit = biasLimit;
```

Comment:
This shouldn't be left uninitialized otherwise you might observe confusing values in the log or worst enable CoM "unbiasing" without setting the limits and get unexpected results

Reply:
Sorry I don't get it, isn't it initialized in line 136?

Reply:
Sorry, it looks like I missed the right-hand part when I read the code x_x